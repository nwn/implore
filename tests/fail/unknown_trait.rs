use implore::impl_op;

#[derive(Copy, Clone, PartialEq, Eq, Debug)]
struct X;

#[impl_op]
fn pow(lhs: X, exponent: u32) -> X {
    lhs
}

fn main() {}
