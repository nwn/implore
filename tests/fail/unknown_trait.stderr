error: unknown operation: `pow`
 --> $DIR/unknown_trait.rs:7:4
  |
7 | fn pow(lhs: X, exponent: u32) -> X {
  |    ^^^
